src/parsers/utils.hpp

```diff
@@ -52,6 +80,10 @@ namespace se3
     // else return an empty string that will provoke an error in loadPolyhedronFromResource()
     namespace bf = boost::filesystem;
 
+    // In case the mesh_path is such as "file://path/to/mesh", check for /path/to/mesh and return it if it exists
+    if ( bf::exists( bf::path(urdf_mesh_path.substr(6, urdf_mesh_path.size())) ))
+      return std::string(urdf_mesh_path.substr(6, urdf_mesh_path.size()));
```

Comment:
I do agree with all those remarks. 
@jmirabel Can you provide some URDF model names with those particularities (file instead of package, etc)?